sampling_time: 0.1
mass: 536.0
gravity: 9.81
fluid_density: 1025.0

# The bounding ellipsoid parameters (ax, ay, az) used for buoyancy calculation
bounding_ellipsoid: [0.5, 0.5, 0.5]

# The dynamic parameters
rg: [0.0, 0.0, 0.0] # The location vector of the COG
rb: [0.0, 0.0, 0.0] # The location vector of the buoyancy 
inertia_matrix: [25,50,50] # Matrix (3x3) or diagonal elements (1x3)
added_mass: [30, 80, 80, 15, 30, 30] # Matrix (6x6) or diagonal elements (1x6)
linear_damping: [100,100,100,400,300,300] # Matrix (6x6) or diagonal elements (1x6)
quadratic_damping: [0,0,0,0,0,0] # Matrix (6x6) or diagonal elements (1x6)

# The initial states
eta0: [0,0,2,0,0,0] # Initial pose
nu0:  [0,0,0,0,0,0] # Initial speeds
current: [0,0,0] # Initial currents (N,E,D)

# The process and measurement noise specification
process_noise: [0.05,0.05,0.05,0.02,0.02,0.02] # Vector (6)
measurement_noise: [1,1,0.02,0.01,0.01,0.01] # Vector (6)
coupled: false # Use a coupled model



